src/test/java/uk/nhs/prm/e2etests/performance/PerformanceTest.java

```diff
@@ -60,54 +73,56 @@ public PerformanceTest(
             ResourceConfiguration resourceConfiguration,
             SuspensionServiceMofUpdatedQueue suspensionServiceMofUpdatedQueue,
             NhsProperties nhsProperties,
-            PdsAdaptorService pdsAdaptorService
+            PdsAdaptorService pdsAdaptorService,
+            TemplatingService templatingService
     ) {
         this.meshMailbox = meshMailbox;
         this.testConfiguration = testConfiguration;
         this.nhsNumbers = resourceConfiguration.nhsNumbers();
         this.suspensionServiceMofUpdatedQueue = suspensionServiceMofUpdatedQueue;
         this.nhsProperties = nhsProperties;
         this.pdsAdaptorService = pdsAdaptorService;
+        this.templatingService = templatingService;
     }
 
     @Disabled("only used for perf test development not wanted on actual runs")
     @Test
```

Comment:
_Really_ splitting hairs here 😂 - can we move the `@Disabled` annotation to under `@Test`?

```suggestion
    @Test
    @Disabled("This test is only used for perf test development not needed on actual runs.")
```